climate-uploader.py

```diff
@@ -0,0 +1,257 @@
+from dotenv import load_dotenv
+import requests
+import json
+import pandas as pd
+import numpy as np
+import argparse
+import os
+from functools import partial
+import sys
+import netCDF4
+from netCDF4 import Dataset
+import xarray as xr
+from matplotlib import pyplot as plt
+import glob
+np.set_printoptions(threshold=np.inf)
+
+load_dotenv()
```

Comment:
Include a template .env (`template.env`) or include instructions in readme on what the .env should contain.